string description
int64 element
float64 speed_now
float64 int_rec
float64 goal_rec
float64 dec_req